http/HttpCache.h

```diff
@@ -97,6 +97,7 @@ namespace http {
         virtual std::string get_cache_file_name(const std::string &src, bool mangle=true);
 #endif
 
+        static unsigned long long getCacheExpiresTime();
```

Comment:
unsigned long long --> unsigned long here too. Also, I'm not sure I understand why this method is 'static'.

Reply:
its static because i just copy/pasted the other method that retrieves the size and changed it to suit